utilities/titandb/blob_file_reader.cc

```diff
@@ -1,11 +1,39 @@
 #include "utilities/titandb/blob_file_reader.h"
 
+#include "util/crc32c.h"
+#include "utilities/titandb/util.h"
+
 namespace rocksdb {
 namespace titandb {
 
 const uint64_t kMinReadaheadSize = 4 << 10;
 const uint64_t kMaxReadaheadSize = 256 << 10;
 
+// Represents a buffer cached in the blob cache.
+class BlobBuffer {
+ public:
+  Slice slice() const { return Slice(data_.get(), size_); }
+
+  size_t cache_size() const { return size_ + sizeof(size_); }
+
+  void assign(std::unique_ptr<char[]> data, size_t size) {
```

Comment:
Maybe change `std::unique_ptr<char[]>` data to `std::unique_ptr<char[]>&&` will make developer more clearly

Reply:
I prefer the current declaration, here are some references:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#r32-take-a-unique_ptrwidget-parameter-to-express-that-a-function-assumes-ownership-of-a-widget
https://stackoverflow.com/questions/8114276/how-do-i-pass-a-unique-ptr-argument-to-a-constructor-or-a-function